CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Cookie Monster Change Log
 
+## 0.1.1
```

Comment:
This contains some changes worthy of a minor bump, `0.2.0`.  We should enumerate all of the significat changes here.